### PR TITLE
feat(blog): Rails getting started.

### PR DESCRIPTION
**Pull request by thorwebdev**
Closes #20835

- [x] Add correct YT video id
- [x] Add correct OG image

Preview: https://zone-www-dot-com-git-thor-blog-rails-supabase.vercel.app/blog/ruby-on-rails-postgres

### CODE REVIEW

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: Getting started with Ruby on Rails and Postgres
```
Comment (kiwicopple) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:2]: should we add "supabase" to the title? 

I'm sure that a lot of peopel will want to search for "using supabase with RoR" at some point

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+description: 'Learn how to create a new Rails app and connect it to a Supabase PostgreSQL database.'
```
Comment (kiwicopple) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:3]:
```suggestion
description: 'Learn how to create a new Rails app and connect it to a Supabase Postgres database.'
```

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  - postgres
+  - developers
+  - ruby
+date: '2024-01-29'
+toc_depth: 3
+---
+
+Every Supabase project comes with a full [Postgres](https://www.postgresql.org/) database, a free and open source database which is considered one of the world's most stable and advanced databases.
+
+PostgreSQL is an ideal choice for your Ruby on Rails applications as Rails ships with a Postgres adapter built in!
```
Comment (kiwicopple) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:19]:
```suggestion
Postgres is an ideal choice for your Ruby on Rails applications as Rails ships with a Postgres adapter built in!
```

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+```bash Terminal
+bin/rails server
+```
+
+Run the development server. Go to http://127.0.0.1:3000 in a browser to see your application running.
+
+## Conclusion
+
+Supabase is the ideal platform for powering your Postgres database for your Ruby on Rails applications! Every Supabase project comes with a full Postgres database and a good number of [useful extension](/docs/guides/database/extensions)!
```
Comment (kiwicopple) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:101]:
```suggestion
Supabase is the ideal platform for powering your Postgres database for your Ruby on Rails applications! Every Supabase project comes with a full Postgres database and a good number of [useful extensions](/docs/guides/database/extensions)!
```

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx (hunk 1 above)
Comment (awalias) [reply apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:2]: what about "Getting started with Ruby on Rails and Supabase Postgres" to capture both?

Comment (awalias) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:2]:
```suggestion
title: Getting started with Ruby on Rails and Postgres on Supabase
```

**Review by awalias — APPROVED** [review]
optional addition bit here from copple:
"any thouts on adding Deployments to this post 
[@Thor](https://supabase.slack.com/team/U019T6ER8KB)
? If you deploy to Fly then I can get their marketing team to tweet it"

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+### Provision Rails with fly launch
+
+To configure and launch the app, you can use `fly launch` and follow the wizard.
+
+When asked "Do you want to tweak these settings before proceeding?" select `y` and set Postgres to `none` as we will be providing the Supabase database URL as a secret.
+
+### Set the connection string as secret
+
+Use the fly CLI to set the Supabase database connection URI from above as a sevret which is exposed as an environment variable to the Rails app.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:124]:
```suggestion
Use the Fly.io CLI to set the Supabase database connection URI from above as a sevret which is exposed as an environment variable to the Rails app.
```

File: apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  # Defines the root path route ("/")
+  root "articles#index"
+end
+```
+
+## Deploy to Fly.io
+
+In order to start working with Fly.io, you will need `flyctl`, our CLI app for managing apps. If you've already installed it, carry on. If not, hop over to the [installation guide](https://fly.io/docs/hands-on/install-flyctl/). Once that's installed you'll want to [log in to Fly](https://fly.io/docs/getting-started/log-in-to-fly/).
+
+### Provision Rails with fly launch
```
Comment (kiwicopple) [inline apps/www/_blog/2024-01-29-ruby-on-rails-postgres.mdx:116]:
```suggestion
### Provision Rails with Fly.io
```

**Pull request merged**